src/common/dsp/SurgeVoice.h

```diff
@@ -61,6 +62,7 @@ class alignas(16) SurgeVoice
     void legato(int key, int velocity, char detune);
     void switch_toggled();
     void freeAllocatedElements();
+    void copyParametersToLocal();
```

Comment:
It all looks really great except for this one added function which I can't find either used or defined. Code booger?

Reply:
> It all looks really great except for this one added function which I can't find either used or defined. Code booger?

Yeah, I missed that one. I used it in a previous attempt where tried to copy the localdata at the voice level. I then realised that it wouldn't work as the scene modulations would already have been copied over. 

I'll remove it

Reply:
Thanks!